.changeset/quiet-starfishes-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-home': patch
+---
+
+Now, user can provide a language (optional) as input to receive a greeting in that specific language. Example: <WelcomeTitle language={['English' 'Spanish']} />
```

Comment:
Small optional change: maybe put backticks (`) around the example so it shows as a code block?

Reply:
fixed